clients/docs/scala.rst

```diff
@@ -0,0 +1,174 @@
+.. _client-examples-scala:
+
+Scala
+======
+
+In this tutorial, you will run a Scala client application that produces messages
+to and consumes messages from an |ak-tm| cluster. For more information, please
+see the `application development documentation
+<https://docs.confluent.io/current/api-javadoc.html?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud>`__
+
+.. include:: includes/client-example-overview.rst
+
+
+Prerequisites
+-------------
+
+Kafka Cluster
+~~~~~~~~~~~~~
+
+.. include:: includes/client-example-prerequisites.rst
+
+
+Setup
+-----
+
+#. .. include:: includes/clients-checkout.rst
+
+#. Change directory to the example for Scala.
+
+   .. code-block:: bash
+
+      cd clients/cloud/scala/
+
+#. .. include:: includes/client-example-create-file-java.rst
+
+
+Basic Producer and Consumer
+---------------------------
+
+.. include:: includes/producer-consumer-description.rst
+
+
+Consume Records
+~~~~~~~~~~~~~~~
+
+#. Run the example consumer:
```

Comment:
This step is unnecessary because the user already when to this directory in the previous step

Reply:
Updated